lib/liquid/combinators/markup.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Liquid.Combinators.Markup do
```

Comment:
This is not a render task, is a transformation from an AST to another AST

Reply:
As we discussed, the converting from new AST to old AST should happen in a separate layer, that could later be thrown away. I would like it to be in a separate folder and namespace so that we can isolate all things that are concerned with that transformation to one area that we can simply delete.

Reply:
You're right, I move it to a new namespaces `Translators`